components/bt/host/bluedroid/api/include/api/esp_gap_ble_api.h

```diff
@@ -17,2687 +17,3936 @@
+     * This event is triggered when the Physical Layer (PHY) update operation is complete for a connection.
      */
     struct ble_phy_update_cmpl_param {
-        esp_bt_status_t status;             /*!< phy update status */
-        esp_bd_addr_t bda;                  /*!< address */
-        esp_ble_gap_phy_t tx_phy;           /*!< tx phy type */
-        esp_ble_gap_phy_t rx_phy;           /*!< rx phy type */
-    } phy_update;                           /*!< Event parameter of ESP_GAP_BLE_PHY_UPDATE_COMPLETE_EVT */
+        esp_bt_status_t status;             /*!< Status of the PHY update operation. Indicates whether the update was successful or if an error occurred. */
+        esp_bd_addr_t bda;                  /*!< Address of the device for which the PHY update was performed. */
+        esp_ble_gap_phy_t tx_phy;           /*!< Transmit PHY type after the update. Indicates the PHY used for transmitting data. */
+        esp_ble_gap_phy_t rx_phy;           /*!< Receive PHY type after the update. Indicates the PHY used for receiving data. */
+    } phy_update;    /*!< ESP_GAP_BLE_PHY_UPDATE_COMPLETE_EVT */
+
     /**
      * @brief ESP_GAP_BLE_EXT_ADV_REPORT_EVT
+     * Event parameter for the ESP_GAP_BLE_EXT_ADV_REPORT_EVT event.
+     * This event is triggered when an extended advertising report is received.
      */
     struct ble_ext_adv_report_param {
-        esp_ble_gap_ext_adv_report_t params;   /*!< extend advertising report parameters */
-    } ext_adv_report;                          /*!< Event parameter of ESP_GAP_BLE_EXT_ADV_REPORT_EVT */
+        esp_ble_gap_ext_adv_report_t params;        /*!< Parameters of the extended advertising report. Contains details about the extended advertising packet received. */
+    } ext_adv_report;   /*!< ESP_GAP_BLE_EXT_ADV_REPORT_EVT */
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_REPORT_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_REPORT_EVT event.
+     * This event is triggered when a periodic advertising report is received.
      */
     struct ble_periodic_adv_report_param {
-        esp_ble_gap_periodic_adv_report_t params; /*!< periodic advertising report parameters */
-    } period_adv_report;                          /*!< Event parameter of ESP_GAP_BLE_PERIODIC_ADV_REPORT_EVT */
-#endif // #if (BLE_50_FEATURE_SUPPORT == TRUE)
+        esp_ble_gap_periodic_adv_report_t params;   /*!< Parameters of the periodic advertising report. Contains details about the periodic advertising packet received. */
+    } period_adv_report;  /*!< ESP_GAP_BLE_PERIODIC_ADV_REPORT_EVT */                   
+#endif// #if (BLE_50_FEATURE_SUPPORT == TRUE)
+
+/* if Bluetooth Low Energy Feature Periodic Advertising Synchronization Transfer Support */ 
 #if (BLE_FEAT_PERIODIC_ADV_SYNC_TRANSFER == TRUE)
+
     /**
      * @brief ESP_GAP_BLE_PERIODIC_ADV_RECV_ENABLE_COMPLETE_EVT
+     * Event parameter for the ESP_GAP_BLE_PERIODIC_ADV_RECV_ENABLE_COM
```

Comment:
esp_ble_gap_update_conn_params
 * @param[in]       params   -  new connection parameters